cmd/istioctl/main.go

```diff
@@ -40,10 +38,51 @@ import (
 	"istio.io/manager/platform/kube"
 )
 
+const (
+	defaultIstioNamespace = "default" // istio-system?
+)
+
+type k8sRESTRequester struct {
+	ri        rest.Interface
+	namespace string
+	version   string
+	port      int
+}
+
+// Construct service proxy path
+// (see https://kubernetes.io/docs/concepts/cluster-administration/access-cluster/#discovering-builtin-services)
+func (r *k8sRESTRequester) absPath(string path) string {
+	return fmt.Sprintf("api/v1/namespaces/%s/services/istio-manager:%d/proxy/%s/%s",
+		r.namespace, r.port, r.version, path)
```

Comment:
Good point. I generalized `apiServerPort` to `managerAPIService` which includes service name and port, e.g. `istio-manager:8081`